### PR TITLE
adig: add page

### PR DESCRIPTION
**Pull request by shubham-kshetre**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/adig.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display A (default) record from DNS for hostname(s):
+
+`adig {{example.com}}`
+
+- Display extra debugging output:
+
+`adig -d {{example.com}}`
+
+- Connect to specified DNS server:
```
Comment (kbdharun) [inline pages/linux/adig.md:14]:
```suggestion
- Connect to [s]pecified DNS server:
```

File: pages/linux/adig.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# adig
+
+> Prints information received from Domain Name System (DNS) servers.
+> More information: <https://manned.org/adig>.
+
+- Display A (default) record from DNS for hostname(s):
+
+`adig {{example.com}}`
+
+- Display extra debugging output:
```
Comment (kbdharun) [inline pages/linux/adig.md:10]:
```suggestion
- Display extra [d]ebugging output:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, Try to use mnemonics whenever possible for single-character flags.

> We don't need to use it in the last 2 commands because they are protocols.

**Review by kbdharun — APPROVED** [review]
LGTM

**Review by blueskyson — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**